configure-lb-healthcheck.html.md.erb

```diff
@@ -35,16 +35,28 @@ in the **Networking** pane of the <%= vars.app_runtime_abbr %> and <%= vars.segm
 To configure your load balancer to use HTTPS health check endpoints, add the IP addresses of all router instances,
 along with their corresponding port and path:
 
+<% if vars.platform_code == 'CF' %>
```

Comment:
These conditionals are no longer needed because we are not sharing branches with the CF docs anymore.
But I think I would like to remove them in a systematic way when I have time to think quietly about each one.
Leaving them for now.